context/app/static/js/components/entity-search/SearchWrapper/SearchWrapper.jsx

```diff
@@ -35,14 +36,25 @@ function SearchWrapper({ uniqueFacets, uniqueFields, entityType }) {
     },
   ];
 
-  const config = { initialFacets, initialFields, facets: initialFacets, fields: initialFields, filters, entityType };
-
+  const numericFacetsProps = useNumericFacetsProps(entityType);
   return (
-    <SearchConfigProvider initialConfig={config}>
-      <SearchkitProvider client={skClient}>
-        <Search />
-      </SearchkitProvider>
-    </SearchConfigProvider>
+    Object.keys(numericFacetsProps).length > 0 && (
```

Comment:
Slight preference to pull this out as a guard clause: Something like:
```
if (!Object.keys(numericFacetsProps).length) return;
return (<SearchConfigProvider 
... 
/>);

Reply:
Thanks!